bin/container.sh

```diff
@@ -143,8 +143,28 @@ start_wasabi() {
   start_docker
 
   id=$(fromPom modules/main development application.name)
-  wcip=$(docker inspect --format "{{ .NetworkSettings.Networks.${docker_network}.IPAddress }}" ${project}-cassandra)
-  wmip=$(docker inspect --format "{{ .NetworkSettings.Networks.${docker_network}.IPAddress }}" ${project}-mysql)
+
+  CASSANDRA_CONTAINER_NAME=${CASSANDRA_CONTAINER:-${project}-cassandra}
+  IS_CONTAINER=`docker inspect -f {{.State.Running}} $CASSANDRA_CONTAINER_NAME`
+  if [ "$IS_CONTAINER" = true ] ; then
+    wcip=$(docker inspect --format "{{ .NetworkSettings.Networks.${docker_network}.IPAddress }}" ${CASSANDRA_CONTAINER_NAME})
+  elif [ -z ${NODE_HOST} ]; then
+    echo "[ERROR] Cassandra container must be running or NODE_HOST environment variable must be set"
+    exit 1;
+  else
+    wcip=${NODE_HOST}
```

Comment:
We could, but semantically CASSANDRA_CONTAINER and NODE_HOST are intended for different things (container which we derive a host from vs a host value itself) . I would prefer to keep them as different variables